src/components/NavBar/NavbarLayout.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <div>
+    <v-app-bar app clipped-left height="70vh">
+      <v-app-bar-nav-icon
+        class="hamburger-icon"
+        @click.stop="drawer = !drawer"
+      ></v-app-bar-nav-icon>
+
+      <v-spacer />
+      <v-app-title class="navbar-title">OpenSlot Studio</v-app-title>
+      <v-avatar>
+        <img src="../../assets/git.svg" alt="Git Logo" />
+      </v-avatar>
+      <v-app-title class="navbar-title">Git Documentation</v-app-title>
+      <v-spacer />
+    </v-app-bar>
+
+    <v-navigation-drawer v-model="drawer" app clipped class="navigation-drawer">
+      <v-list nav>
+        <v-list-item-group active-class="actived-list-item">
+          <v-list-item to="/">
+            <v-list-item-content>
+              <v-list-item-title>Description</v-list-item-title>
+            </v-list-item-content>
+          </v-list-item>
+          <v-list-item to="/git-basics">
+            <v-list-item-content>
+              <v-list-item-title>Git Basics</v-list-item-title>
+            </v-list-item-content>
+          </v-list-item>
+          <v-list-item to="/git-workflow">
+            <v-list-item-content>
+              <v-list-item-title>Git Workflow</v-list-item-title>
+            </v-list-item-content>
+          </v-list-item>
+          <v-list-item to="git-rules">
+            <v-list-item-content>
+              <v-list-item-title>Git Rules</v-list-item-title>
+            </v-list-item-content>
+          </v-list-item>
+        </v-list-item-group>
+      </v-list>
+    </v-navigation-drawer>
+  </div>
+</template>
```

Comment:
@JuanJosePG I would like have a little meeting with you to talk about this, because here I see that in the navbar you have define the navbar and the sidebar, and I don´t know if this is another method to create and define the sidebar and the navbar. I think that is better separe the navbar of the sidebar, but if this is another method to define the sidebar I would like learn.

Reply:
Okay, let's have the meeting the next week. I have the sidebar and the navbar together because I didn't realise when I wrote it. For me, it's fine because I don't have two simple components separated and somehow one affects the other and I prefer to have them together.